spec/takeaway_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'Takeaway'
+
+describe Takeaway do
+
+  subject(:takeaway1) {described_class.new}
+  order_hash = {"chicken noodle soup" => 2,
+           "burger and fries" => 1,
+           "chicken tikka madras" => 4,
+           "fresh caesar salad" => 1}
+
+  wrong_order = {"chicken noodle soup" => 1,"chicken tikka madras" => 1, 'beans on toast' => 1}
+
+  it 'creates a new Takeaway' do
+    expect(described_class.new).to be_a(Takeaway)
+  end
+
+  it 'has an order method' do
+    expect(subject).to respond_to(:place_order).with(2).arguments
+  end
+
+  it 'takes an order in form of a hash' do
+    allow(subject).to receive(:sms_send).and_return(true)
+    subject.place_order(order_hash,46.5)
+    expect(subject.customer_order).to be_a (Hash)
+    end
```

Comment:
Lint/BlockAlignment: `end` at 25, 4 is not aligned with `it 'takes an order in form of a hash' do` at 21, 2.